sections/GPU.tex

```diff
@@ -1,130 +1,19 @@
-%METADATA about GPU, purpose and general difference from CPU
 \section{Graphics Processing Units}
-Graphic Processing Units (GPUs) were initially used to accelerate memory-intensive work for rendering graphics.
-They consist of a higher amount of cores, these cores while each less powerfull than the cores used in a CPU, are so plentiful that with enough work, more data can be processed in parallel.
-These cores sacrifice the architectural components that enables the CPU run single instruction streams in exchange for more computing power, such as increasing the number of \textbf{Arithmetic Logic Units (ALUs)} \myref{image:GPUCPUimage} shows a basic idea of the difference between the GPU and CPU. %http://pdsgroup.hpclab.ceid.upatras.gr/files/CUDA-Parallel-Computing-on-GPUs.pdf
+Dedicated Graphic Processsing Units (``GPUs'') are very common in desktop and laptops. \citep{STEAMHW}
+GPUs are commonly used in the private and professional world, for computer gaming and accelerating graphic intensive programs such as Adobe Photoshop and 3D modeling tools. \citep{NVIDIAADOBE}
+Their architecture can also be used for certain types of computations, primarily computations which can be done in parallel. 
```

Comment:
Obviously they can be used for computations, "[...] be used advantageously [...]